.github/workflows/snapshot-build.yml

```diff
@@ -90,6 +90,12 @@ jobs:
             latest=false
             suffix=
 
+      # Use the same length as the fallback command in the substrate build script:
+      # https://github.com/autonomys/polkadot-sdk/blob/master/substrate/utils/build-script-utils/src/version.rs#L28
```

Comment:
Please use permalink here

Reply:
I don't think we need the latest code here, we just need to show what it corresponds to and we can always update the reference. Pointing to a moving target is less helpful and frustrating when things shift around/disappear.

BTW `master` of our fork is essentially identical to upstream (just lagging behind).

Reply:
Done!